phys2bids/viz.py

```diff
@@ -41,3 +41,21 @@ def ntr2time(x):
     subplot.plot(time, trigger, '-', time, time, '-')
     plt.savefig(outfile + '_trigger_time.png', dpi=dpi)
     plt.close()
+
+def plot_all(phys_in,infile,outfile,dpi = SET_DPI,size = FIGSIZE):
+    ch_num = len(phys_in.ch_name) # get number of channels:
+    fig,ax = plt.subplots(ch_num-1,1,figsize = size,sharex=True)
+    row = 0
+    time = phys_in.timeseries[0] # assume time is first channel
+    fig.suptitle(infile)
```

Comment:
Why don't we return just the filename?
Also, it would be great if we can make this a big bigger (2 times the size it is now).
```suggestion
    fig.suptitle(os.path.basename(infile))
```